lib/collection/response.js

```diff
@@ -292,6 +293,24 @@ _.assign(Response.prototype, /** @lends Response.prototype */ {
         return LJSON.parse(this.text(), reviver, strict);
     },
 
+    /**
+    * Extracts mime info and filename of the response content
+    * A fallback of default filename is given, if filename is not present in header
+    *
+    * @returns {Object} - Return contentInfo of the response
```

Comment:
Need to document the return type. A TypeDef?

Reply:
yes `ResponseContentInfo` typedef